CreateExeOfConsoleApp.cmd

```diff
@@ -0,0 +1,13 @@
+
+SET outputDir=.\ExecutableFiles
+SET exeFileName=Squirrel.ConsoleApp
+SET exeCustomFileNamePrefix=Squirrel
+SET pathToSln=.\backend\Squirrel.ConsoleApp.sln
+
+del /s /q "%outputDir%"
+
+dotnet publish "%pathToSln%" -r win-x64 -p:PublishSingleFile=true -p:IncludeNativeLibrariesForSelfExtract=true -p:PublishTrimmed=true -p:EnableCompressionInSingleFile=true --output "%outputDir%"
+ren "%outputDir%\%exeFileName%.exe" "%exeCustomFileNamePrefix%-win-x64.exe"
```

Comment:
Maybe we can use a subroutine to reduce code duplication, for example:
```
call :PublishApp win-x64
call :PublishApp win-x86
exit /b

:PublishApp
set architecture=%1

dotnet publish "%pathToSln%" -r %architecture% -p:PublishSingleFile=true -p:IncludeNativeLibrariesForSelfExtract=true -p:PublishTrimmed=true -p:EnableCompressionInSingleFile=true --output "%outputDir%"
ren "%outputDir%\%exeFileName%.exe" "%exeCustomFileNamePrefix%-%architecture%.exe"
```

Reply:
Totally agree, thanks!